boberfoell.md

```diff
@@ -0,0 +1,5 @@
+# Bob Oberfoell
+
+I teach in Dubuque, IA.
```

Comment:
I hung out in Cedar Rapids for a youth Min Internship, not sure how far that was from you. Cool state!